docs/developing/README.md

```diff
@@ -1,13 +1,22 @@
 # Local setup
 
-Run these commands:
+Follow these instructions to run the app locally:
 
-```
-docker-compose up -d postgres
-docker-compose run app /bin/bash -c "npx pubsweet setupdb --username=foo --password=password --email=foo@example.com --clobber"
-docker-compose up
-```
+[PubSweet development setup](https://gitlab.coko.foundation/pubsweet/pubsweet/wikis/Development:-setup)
 
 Visit the web interface at [http://localhost:3000].
 
-> **TODO** Documentation for styleguide
\ No newline at end of file
+> **TODO** Documentation for styleguide
+
+### Dependencies
+
+The application depends on S3. You can run a fake S3 server locally:
+- Run `docker run -p 4569:4569 --rm lphoward/fake-s`
+- Add the following to `config/local.js`
```

Comment:
I think `config/local-development.js` might be a better suggestion or else it will override the test config causing the tests to fail.